Package.swift

```diff
@@ -30,14 +30,12 @@ let package = Package(
         "AirbnbSwiftFormatTool",
         .product(name: "swiftformat", package: "SwiftFormat"),
         .product(name: "swiftlint", package: "SwiftLint"),
-      ],
-      path: "AirbnbSwiftFormatPlugin"),
```

Comment:
We can remove this path by putting the source in a folder under the default Plugins directory.